dallinger/config.py

```diff
@@ -25,6 +24,32 @@ def is_valid_json(value):
     json.loads(value)
 
 
+def strtobool(val):
```

Comment:
If we agree on always adding type hints and docstrings (Parameters, Returns, ...), this could be added here. Otherwise approved.

Reply:
OK added